scrapy/core/downloader/tls.py

```diff
@@ -48,6 +48,11 @@ def _identityVerifyingInfoCallback(self, connection, where, ret):
                         'Remote certificate is not valid for hostname "{}"; {}'.format(
                             self._hostnameASCII, e))
 
+                except ValueError as e:
+                    logger.warning(
+                        'SSL/TLS verification failed for hostname "{}"; {}'.format(
+                            self._hostnameASCII, e))
```

Comment:
@redapple I have 2 reservations about the error message:

1. 'certificate verification failure' can be read as 'wrong certificate' or a 'misconfigured remote website' or 'wrong configuration of local verification software'. But here it is not a bad certificate or old openssl or a problem with a remote website, it is just a bug in software; because of a bug the verification is not even attempted. I don't know how to express that well in error message though.
2. it says `'hostname {}'.format(self._hostnameASCII)`, but _hostnameASCII is not a hostname here - it is an IP, and this is precisely what causes the error. 

I think we shouldn't describe the actual bug in the error message because ValueError is too broad - it could happen other issue will cause ValueError in future, or maybe it can be triggered now by something else.

What about `"Ignoring certificate verification error for {}; {}".format(self._hostnameASCII, e)`? It doesn't fully address (1).